From 60813c7f5f1ceff4cc722bc34e2f8594e49c76e6 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Thu, 31 Jan 2019 12:54:12 +0900
Subject: [PATCH 2/5] docs: update status command of primary

This patch is to update status sub command of pri to include lcores.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 docs/guides/commands/primary.rst | 21 ++++++++++++---------
 1 file changed, 12 insertions(+), 9 deletions(-)

diff --git a/docs/guides/commands/primary.rst b/docs/guides/commands/primary.rst
index 6742d28..11e378c 100644
--- a/docs/guides/commands/primary.rst
+++ b/docs/guides/commands/primary.rst
@@ -10,19 +10,22 @@ Primary process is managed with ``pri`` command.
 status
 ------
 
-Show forwarding statistics of each of ports.
+Show status fo spp_primary and forwarding statistics of each of ports.
 
 .. code-block:: console
 
     spp > pri; status
-    Physical Ports:
-      ID          rx          tx     tx_drop  mac_addr
-       0    78932932    78932931           1  56:48:4f:53:54:00
-    Ring Ports:
-      ID          rx          tx     rx_drop     tx_drop
-       0       89283       89283           0           0
-       1        9208        9203           0           5
-       ...
+    - lcores:
+      - [0]
+    - physical ports:
+        ID          rx          tx     tx_drop  mac_addr
+         0           0           0           0  56:48:4f:53:54:00
+         1           0           0           0  56:48:4f:53:54:01
+    - ring Ports:
+        ID          rx          tx     rx_drop     rx_drop
+         0           0           0           0           0
+         1           0           0           0           0
+
 
 clear
 -----
-- 
2.7.4

